src/Loader/SymfonyBridgeLoader.php

```diff
@@ -5,18 +5,8 @@
 namespace Doctrine\Bundle\FixturesBundle\Loader;
 
 use Doctrine\Common\DataFixtures\Loader;
-use Symfony\Bridge\Doctrine\DataFixtures\ContainerAwareLoader;
 
-use function class_exists;
-
-if (class_exists(ContainerAwareLoader::class)) {
-    /** @internal */
-    abstract class SymfonyBridgeLoader extends ContainerAwareLoader
-    {
-    }
-} else {
-    /** @internal */
-    abstract class SymfonyBridgeLoader extends Loader
-    {
-    }
+/** @internal */
+abstract class SymfonyBridgeLoader extends Loader
+{
```

Comment:
The internal `SymfonyBridgeLoader` class was introduced because we needed to conditionally extend the `ContainerAwareLoader` in order to not break BC. Leaving this as an empty class is pretty pointless.

This class has been removed on the 4.0.x branch already.